src/components/nested_screens/profile/TradeHistoryScreen.js

```diff
@@ -1,10 +1,109 @@
-import React from "react";
```

Comment:
All the logic code for getting the data for the table that was added to this file should be placed in the `CustomTable` component so that it can be re used throughout the app. For example if we want to render a `CustomTable` from the job details screen we would have no reference to any of this code that was written so we wouldn't be able to render the table.